modules/Template.js

```diff
@@ -0,0 +1,19 @@
+function li1(title) {
+  const li = document.createElement('li');
+  li.className = 'book-title';
+  li.innerHTML = `"${title}" by`;
+  return li;
+}
+
+function li2(author) {
+  const li = document.createElement('li');
+  li.className = 'book-author';
+  li.innerHTML = author;
+
+  return li;
+}
```

Comment:
- [ ] Same goes here ☀️, please make sure to use arrow syntax for functions

`const li1 = (title) => { logic }`